example/opensourcecmf/opensource_replica_msgs.cmf

```diff
@@ -0,0 +1,62 @@
+# Definition of client-server interface for the opensource example test application.
+#
+# Note OSRKVBC effectively implements a versioned key-value store supporting
+# only keys and values that are byte strings.
+
+Msg OSRKVBCReadRequest 3 {
+
+    # Note an OSRKVBCReadRequest with read version 0 indicates that the latest
+    # version should be read.
+    uint64 read_version
+
+    list bytes keys
+}
+
+Msg OSRKVBCWriteRequest 4 {
+    uint64 read_version
+
+    # if set to true, the Open Source replica KVBC will waste some time when it works
+    # on pre-processing this request, which may be useful for simulating certain
+    # workloads in testing.
+    bool long_exec
+
+    list bytes readset
+    list kvpair bytes bytes writeset
+}
+
+Msg OSRKVBCGetLastBlockRequest 5 {
+}
+
+Msg OSRKVBCGetBlockDataRequest 6 {
+    uint64 block_id
+}
+
+Msg OSRKVBCReadReply 7 {
+    list kvpair bytes bytes reads
+}
+
+Msg OSRKVBCWriteReply 8 {
+    bool success
+    uint64 latest_block
+}
+
+Msg OSRKVBCGetLastBlockReply 9 {
+    uint64 latest_block
+}
+
+Msg OSRKVBCRequest 1 {
+    oneof {
+        OSRKVBCReadRequest
+        OSRKVBCWriteRequest
+        OSRKVBCGetLastBlockRequest
+        OSRKVBCGetBlockDataRequest
+    } request
+}
+
+Msg OSRKVBCReply 2 {
+    oneof {
+        OSRKVBCReadReply
+        OSRKVBCWriteReply
+        OSRKVBCGetLastBlockReply
+    } reply
+}
```

Comment:
same here - let's consider replacing the cumbersome `OSRKVBC` prefix with simply `KV`, or `KVBC` - or something better?

Reply:
@teoparvanov we can consider prefix KV for all message commands for this. Thanks for the suggestion .